event/workflow.go

```diff
@@ -1,9 +1,87 @@
 package event
 
+import (
+	"github.com/qri-io/qri/profile"
+)
+
 const (
 	// ETWorkflowTrigger signals that a workflow has been triggered
-	// Payload will be a string representation of the `workflow.ID`
-	// of the workflow that should be run
+	// Payload will be a WorkflowTriggerEvent
```

Comment:
All of these events should be namespaces under `automation` instead of `workflow`. Match constant strings to the existing event convention of `subsystem:UpperCamelCase`